content/about/contributing/contributing.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <title>Contributing to the APG</title>
+
+  <!-- Core JS and CSS shared by all About content -->
+  <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+  <link rel="stylesheet" href="../../shared/css/core.css">
+  <script src="../../shared/js/highlight.pack.js"></script>
+  <script src="../../shared/js/app.js"></script>
+  <script src="../../shared/js/skipto.js"></script>
+</head>
+
+<body>
+  <main>
+    <h1>Contributing to the APG</h1>
+
+    <section id="intro">
+      <h2>Introduction  </h2>
+      <p>
+        Continuous improvement to the utility and quality of the APG depends on broad community input and participation.
+        Anyone may contribute feedback, suggestions, or code via the channels described below.
+        However, if you would like to contribute regularly, please consider
+        <a href="#joining-task-force">joining the Authoring Practices Task Force</a>.
+        As a task force participant, you have greater influence over APG development, and you and your organization will be listed as contributors where appropriate.
+      </p>
+    </section>
+
+    <section id="feedback">
+      <h2>Providing feedback</h2>
+      <p>
+        The APG provides two public feedback channels -- GitHub and email.
+        Links to both are provided at the bottom of every page in the APG.
```

Comment:
```suggestion
        Links to both are provided at the bottom of every APG page.
```

Reply:
Left unchanged because so the emphasis is on "every page". "In the APG" is almost superfluous, but I've chosen to leave it because the claim might not be consistently true across all WAI pages.

Reply:
It should be, but I don't feel strongly anyway.